vrx_gazebo/worlds/gymkhana.world.xacro

```diff
@@ -104,6 +105,8 @@
       <contact_debug_topic>/vrx/gymkhana_blackbox/debug/contact</contact_debug_topic>
       <!-- Keep Gazebo running after this sub-task is completed -->
       <per_plugin_exit_on_completion>false</per_plugin_exit_on_completion>
+      <!-- Disable heading error -->
+      <heading_weight>0</heading_weight>
```

Comment:
Consistent with other comments, can we change the name and make the argument for this true/false?

Reply:
Thanks for the suggestions. I changed to a boolean flag in 758c80c. I already started with a boolean flag approach, and then, I changed my mind to expose `k` :) . That's why I left unintentionally the parsing bug. It should be good to go!